### PR TITLE
downgrade to jdk 8

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
The DAML project is still using JDK 8, so this PR makes the code compatible with that. To avoid future breakages, it also downgrades the compiler used on CI to 8.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       daml_sdk_version:
         type: string
     docker:
-      - image: circleci/openjdk:11.0-jdk
+      - image: circleci/openjdk:8-jdk
```
Comment (maxdeliso-da) [inline .circleci/config.yml:30]: please upgrade the readme as well, as it mentions 11

File: src/main/scala/com/projectdabl/authenticationservice/Main.scala [hunk 2]
```diff
@@ -125,7 +125,7 @@ object Main extends LazyLogging {
             rsaKeyPair,
             keyId,
             serviceConfig.jwtConfig.issuer,
-            Duration(serviceConfig.jwtConfig.validityDuration.toSeconds, SECONDS)
+            Duration(serviceConfig.jwtConfig.validityDuration.getSeconds, SECONDS)
```
Comment (maxdeliso-da) [inline src/main/scala/com/projectdabl/authenticationservice/Main.scala:128]: was there really no change required to the config lib in use? or did it automatically produce a non ```java.time``` using value when the jdk in used changed? seems like there should have been a change needed there, too, based on what I recall about the way that library works.

Comment (garyverhaegen-da) [reply src/main/scala/com/projectdabl/authenticationservice/Main.scala:128]: `java.util.Duration` is present in jdk8, it just lacks that one `toSeconds` method. The reason for it is Duration is really just a wrapper for a (seconds, nanoseconds) tuple, so when they first implemented it they had the normal accessors, `getSeconds` and `getNanos`, and then a bunch of converters `toMinutes`, `toHours`, etc.

Then people complained about the lack of regularity in the API, and they added `toSeconds` as an alias to `getSeconds` to uniformize it a bit.

File: .circleci/config.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply .circleci/config.yml:30]: Done.

**Pull request merged**